doc-checking-app-api/src/main/java/uk/gov/di/authentication/app/lambda/DocAppCallbackHandler.java

```diff
@@ -306,13 +287,8 @@ public APIGatewayProxyResponseEvent docAppCallbackRequestHandler(
                 auditService.submitAuditEvent(
                         AUTH_CODE_ISSUED,
                         clientId,
-                        clientSessionId,
-                        session.getSessionId(),
-                        clientSession.getDocAppSubjectId().getValue(),
-                        session.getEmailAddress(),
-                        IpAddressHelper.extractIpAddress(input),
-                        AuditService.UNKNOWN,
-                        AuditService.UNKNOWN,
+                        user.withEmail(session.getEmailAddress())
```

Comment:
I know this isn't changing this, but I think the email address will never be known because they're docapp?

Reply:
Agree - this shouldn't percolate into the txma stream because that value will be null. What's your preference here, make it explicit and not add email at this callsite?

Reply:
I think so, yes. I think it's misleading to be looking for something we know will never be there